### PR TITLE
FMRF-22: Check your answers before sending your request

### PR DESCRIPTION
**Pull request by dk4g**
## What? 
Add the summary section
## Why? 
[FMRF-22](https://collaboration.homeoffice.gov.uk/jira/browse/FMRF-22) - FMR - Check your answers before sending your request
## How? 
Added summary-data-sections.js and necessary parameters to pages.json. Added views/partials
Updated other files to work with summary requirements.
## Anything Else? (optional)
- added content on Request Sent page
- removed duplicated step 'sonar_scanner_branch' from CI/CD pipeline
## Check list

- [ ] I have reviewed my own pull request
- [ ] I have written tests (if relevant)

### CODE REVIEW

File: apps/fmr/sections/summary-data-sections.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+        field: 'someone-else'
+      }
+    ]
+  },
+  'completing-for-someone-else': {
+    steps: [
+      {
+        step: '/someone-else-details',
+        field: 'detail-signin-email'
+      },
```
Comment (jamiecarterHO) [inline apps/fmr/sections/summary-data-sections.js:71]: Is this field in the `/someone-else-details` step?

File: apps/fmr/views/request-sent.html [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{{<partials-page}}
+  {{$pageTitle}}
+    {{#t}}pages.request-sent.confirmation{{/t}} &ndash; {{#t}}journey.serviceName{{/t}} &ndash; GOV.UK
+  {{/pageTitle}}
+  {{$page-content}}
+    <div class="govuk-panel govuk-panel--confirmation" role="alert">
+     <h1 class="govuk-panel__title">
+        {{#t}}pages.request-sent.confirmation{{/t}}
+      </h1>
+    </div>
```
Comment (jamiecarterHO) [inline apps/fmr/views/request-sent.html:10]: I guess the rest of this is added with the reference generation ticket?

**Review by jamiecarterHO — COMMENTED** [review]
Couple questions

File: apps/fmr/sections/summary-data-sections.js (hunk 1 above)
Comment (dk4g) [reply apps/fmr/sections/summary-data-sections.js]: has been removed

File: apps/fmr/views/request-sent.html (hunk 2 above)
Comment (dk4g) [reply apps/fmr/views/request-sent.html:10]: yes, that will be covered separately

**Review by jamiecarterHO — APPROVED** [review]

**Pull request merged**